### PR TITLE
chore: upsert frontend sample env files

### LINKED ISSUE #394: Update sample.env for frontend
## Bug Report

See issue #393 and make the same for the frontend.

Optionally can do the same for Nextjs's env :)

### PR DESCRIPTION
**Pull request by McTechie**
# Pull Request

## What does this PR do?
Fixes #394 

- Added bare minimum comments to guide users in `FrontEndApp/v1-react/sample.env`
- Created `FrontEndApp/autodl-platform/.env.example`

## What part does this affect?
- [x] FrontEnd.
- [ ] BackEnd.
- [ ] Documentation.
- [x] Other. (Please specify below)

Sample Env files present in `FrontEndApp`

## Before submitting
- [x] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [x] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [ ] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

**Review by ADI10HERO — DISMISSED** [review]
Perfetto! ❤️

File: FrontEndApp/autodl-platform/.env.example [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# server url
```
Comment (ADI10HERO) [inline FrontEndApp/autodl-platform/.env.example:1]:
```suggestion
# server url (required)
```

File: FrontEndApp/v1-react/sample.env [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# server url
```
Comment (ADI10HERO) [inline FrontEndApp/v1-react/sample.env:1]:
```suggestion
# server url (required)
```

File: FrontEndApp/autodl-platform/.env.example [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+NEXT_APP_BACKEND_API_URL=http://localhost:8000
+
+# recommendation system url
+NEXT_APP_FASTAPI_URL=http://localhost:8080
+
+# upload to github
+NEXT_APP_GITHUB_APP_CLIENT_ID=abcdoadsnfioasndf343
```
Comment (RusherRG) [inline FrontEndApp/autodl-platform/.env.example:8]: For NextJS it's `NEXT_PUBLIC_` but doing so will make the variable also accessible in the browser. So we don't need any prefixes.
https://nextjs.org/docs/basic-features/environment-variables

**Review by RusherRG — CHANGES_REQUESTED** [review]

File: FrontEndApp/autodl-platform/.env.example (hunk 3 above)
Comment (McTechie) [reply FrontEndApp/autodl-platform/.env.example:8]: Yes I would've removed the prefixes, but felt like keeping it similar to the React one. 😅
I'll remove it right away ~

**Comment by McTechie** [comment]
> Just 1 question, why not sample.env in autodl-platform? @McTechie

Well I did try `sample.env` but Git refused from tracking the file 😆. Later figured out that `.env.example` is the way to go in NextJS.

**Review by ADI10HERO — APPROVED** [review]

**Review by RusherRG — APPROVED** [review]

**Pull request merged**